src/utils/common.ts

```diff
@@ -58,33 +58,42 @@ export function getMDHintLinks(content: string, offset: number, letters: string)
     // expecting either [[Link]] or [[Link|Title]]
     const regExInternal = /\[\[(.+?)(\|.+?)?]]/g;
     // expecting [Title](../example.md)
-    const regExMdInternal = /\[.+?]\(((\.\.|\w|\d).+?)\)/g;
+    const regExMdInternal = /\[[^\[\]]+?\]\((.+?:\/\/.+?)\)/g;
```

Comment:
Regex did match `[[Internal Link]] and [External Link](...) ` as one link.

Reply:
> Regex did match `[[Internal Link]] and [External Link](...) ` as one link.

True, but purposed regexp will not handle `[Title](../example.md)` because it expects a `:` symbol.
This is a helper for an internal links.

Reply:
There is another bug, if link in todo list like `- [ ] [link](obsidian://opengate?title=flomo-review) `  can not be detect as a internal link.

Reply:
@mrjackphil Yeah, that's my bad, i meant to change `regExExternal`. I changed that back.
However i kept the check inside the `regExMdInternal`, to check that there are no brackets inside the brackets. That takes care of the todo list issue @jiyee mentioned

Reply:
Seems fine, thanks!